### ISSUE TITLE
/death and /alldeath do not show death counts.

### ISSUE THREAD

**Issue by Marotheit** [Pryaxis/TShock#2714]
* TShock version: 5.0.0-beta
* TShock build number (if known): d3f542c

#### Screenshots

![Before](https://user-images.githubusercontent.com/26125775/194809668-a3a98c68-9cfb-4d10-a3c5-81b2a8b6022b.png)
![After](https://user-images.githubusercontent.com/26125775/194809727-ac59d65e-ef1f-41fd-8c6b-7521fc21db2f.png)

#### Any log messages from files that end in `.log` or `.txt`? What are the last 100 log messages from the server console?

[Console.log](https://github.com/Pryaxis/TShock/files/9743706/2022-10-10_00-36-52.log)

#### What plugins and what versions of those plugins are you running?

None (besides TShock).

**Comment by Arthri** [comment]
afaict, TShock converts Terraria chat commands back into string form, so it goes through TShock rather than Terraria, but TShock neither has a `/death` nor `/alldeath` command

**Comment by Marotheit** [comment]
> afaict, TShock converts Terraria chat commands back into string form, so it goes through TShock rather than Terraria, but TShock neither has a `/death` nor `/alldeath` command

Makes sense. Either way, it's present in the Terraria Dedicated Server so the functionality should probably make it's way into TShock.
